tests/spread/general/run-user/task.yaml

```diff
@@ -0,0 +1,24 @@
+summary: nonroot run-user test
+
+execute: |
+  run_rockcraft pack
+
+  test -f run-user-test_latest_amd64.rock
+  test ! -d work
+
+  # Ensure docker does not have this container image
+  docker rmi --force run-user-test
+  # Install container
+  sudo /snap/rockcraft/current/bin/skopeo --insecure-policy copy oci-archive:run-user-test_latest_amd64.rock docker-daemon:run-user-test:latest
+  # Ensure container exists
+  docker images run-user-test | MATCH "run-user-test"
+  docker inspect run-user-test --format '{{.Config.User}}' | MATCH "confined_daemon"
+  
+  # ensure container doesn't exist
+  docker rm -f run-user-test-container
+  docker run --rm --entrypoint /bin/sh run-user-test -c 'whoami' | MATCH "confined_daemon"
```

Comment:
what about `docker run --rm run-user-test exec -c 'whoami'`? To ensure that pebble itself is running as `confined_daemon` correctly

Reply:
sure